### PR TITLE
O3653-9: Added Visual Studio 2015 starter and completed projects. Created separate completed project for each lab exercise (where applicable). Updated app -> Add-in text in slide deck. Added new demo readme.  Updated all images. Updated lab.md file screenshots and verbiage. Addresses issues #228, #172, #158, and #117.

### PR DESCRIPTION
**Pull request by TBag**
O3653-9: Added Visual Studio 2015 starter and completed projects. Created separate completed project for each lab exercise (where applicable). Updated app -> Add-in text in slide deck. Added new demo readme.  Updated all images. Updated lab.md file screenshots and verbiage. Addresses issues #228, #172, #158, and #117.

### CODE REVIEW

**Comment by jthake** [comment]
OK I'm getting an error on the completed project

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0234	The type or namespace name 'Clients' does not exist in the namespace 'Microsoft.IdentityModel' (are you missing an assembly reference?)	Office365Group	C:\Users\jthake\git\TbagFork\O3653\O3653-9 Deep dive into the Office 365 Groups API\Completed project\Exercise 4\Office365Group\Office365Group\App_Start\Startup.Auth.cs	12	Active


Basically my project won't build and gets  errors in all classes using Microsoft.IdentityModel.

This Microsoft.IdentityModel.Clients.ActiveDirectory.dll has a warning symbol in the project too.

**Comment by TBag** [comment]
Local file system path was too long.  We shortened the folder name and the merge was committed.
